test/tests/caliptra_integration_tests/smoke_test.rs

```diff
@@ -461,7 +465,8 @@ fn smoke_test() {
         Some(DiceTcbInfo {
             vendor: Some("Caliptra".into()),
             model: Some("RT".into()),
-            svn: Some(0x100),
+            // This is from the SVN in the image (9)
+            svn: Some(0x109),
```

Comment:
Why is there an extra byte?

Reply:
Looks like it's to keep the encoded width constant.